src/test/ui/span/issue-43927-non-ADT-derive.stderr

```diff
@@ -1,33 +1,10 @@
-error[E0774]: `derive` may only be applied to structs, enums and unions
-  --> $DIR/issue-43927-non-ADT-derive.rs:3:1
+error: cannot determine resolution for the attribute macro `derive`
+  --> $DIR/issue-43927-non-ADT-derive.rs:1:4
    |
 LL | #![derive(Debug, PartialEq, Eq)] // should be an outer attribute!
-   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try an outer attribute: `#[derive(Debug, PartialEq, Eq)]`
```

Comment:
It's a shame, this seems like a marginal regression.

Reply:
This is unfortunate. However, this seems very unlikely to come up in practice, and this PR has been sitting open for quite a while. @petrochenkov Could you open an issue to track improving this diagnostic?

Reply:
There's already a very similar issue - https://github.com/rust-lang/rust/issues/67107.